src/ef-json-query-testing/SearchService.cs

```diff
@@ -61,22 +61,18 @@ public List<Media_Json> JsonSearch_Raw(Dictionary<int, string> searchFields)
 
                 parameters.Add($"$.\"{field.JsonName}\"");
 
-                sqlStatement += $" AND JSON_VALUE([Details], {{{count}}}) ";
-                count++;
-
                 if (field.DataType == DataTypes.StringValue)
                 {
-                    var containsString = "%" + searchField.Value + "%";
-                    parameters.Add(containsString);
-                    sqlStatement += $" like {{{count}}}";
+                    sqlStatement += $" AND CHARINDEX(JSON_VALUE([Details], {{{count++}}}), {{{count++}}}) > 0 ";
```

Comment:
dont really think i want these changes. basically just duplicating the ef stuff.